Email forget password

AJadenHuang opened the pull request:
Added the functionality of sending an email for user, so they can click the link in the email to reset their password.

CarloVelarde reviewed (changes requested):
Refer to comment above about removing .env file and adding forgot password to view

AJadenHuang commented:
Yes, you are right. I've already removed the .env file in my latest commit and changed the API password, so I think this PR won't have .env file into main. And I will add forgot password link right away.

AJadenHuang commented:
Thank you for your suggestions and your time to review my code. I've pushed the latest changes for the forgot-password email.

CarloVelarde commented:
> Thank you for your suggestions and your time to review my code. I've pushed the latest changes for the forgot-password email.

Awesome thanks for getting on that! Looks great
